Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -201,6 +201,15 @@ struct PerWebProcessState {
 
 #endif // PLATFORM(IOS_FAMILY)
 
+#if HAVE(UIFINDINTERACTION)
+enum FindOverlayView {
```

Comment:
should this be an enum class : uint8_t ?

Reply:
or maybe it should just be a struct instead of an enum + a loosely-ordered vector? like
```
struct FindOverlayViews {
 RetainPtr<UIView> left;
// etc
}
```